target/linux/mvebu/image/cortexa9.mk

```diff
@@ -232,3 +232,18 @@ define Device/solidrun_clearfog-pro-a1
   BOOT_SCRIPT := clearfog
 endef
 TARGET_DEVICES += solidrun_clearfog-pro-a1
+
+define Device/kobol_helios4
+  DEVICE_VENDOR := Kobol
+  DEVICE_MODEL := Helios4
+  KERNEL_INSTALL := 1
+  KERNEL := kernel-bin
+  DEVICE_PACKAGES := mkf2fs e2fsprogs partx-utils swconfig
+  IMAGES := sdcard.img.gz
+  IMAGE/sdcard.img.gz := boot-scr | boot-img-ext4 | sdcard-img-ext4 | gzip | append-metadata
+  DEVICE_DTS := armada-388-helios4
+  SUPPORTED_DEVICES += armada-388-helios4
```

Comment:
What's that for?

Reply:
I don't know, I've just copied the Clearfog since this device is very similar. Also all other devices in this file have a similar entry.

Reply:
Then just drop this line. It's for adding _additional_ device names, either after a rename or for cross-flashing, which both doesn't apply here. The primary name is created from the device node name.

Reply:
ok